examples/lock-app/efr32/BUILD.gn

```diff
@@ -91,8 +92,7 @@ if (chip_enable_wifi) {
   ]
 
   if (use_rs911x) {
-    wiseconnect_sdk_root =
-        "${chip_root}/third_party/efr32_sdk/wiseconnect-wifi-bt-sdk"
+    wiseconnect_sdk_root = "${chip_root}/third_party/wiseconnect-wifi-bt-sdk"
```

Comment:
this should not have changed either. you are setting the old path here

Reply:
Switched back to old path